doc/api/inspector.md

```diff
@@ -52,6 +52,21 @@ parameter usage.
 
 Return the URL of the active inspector, or `undefined` if there is none.
 
+```console
+$ node --inspect -p "require('inspector').url()"
```

Comment:
Since built-in modules are automatically loaded when Node.js is run this way, we could consider simplifying to something like this here and below?:

```suggestion
$ node --inspect -p 'inspector.url()'
```

Reply:
And adding to the suggested change above, we can make the output easier to read by making it clear what part of the output is from `inspector.url()` since there are a couple other lines of output that might confuse the reader?:

```suggestion
$ node --inspect -p '`\ninspector.url() result: ${inspector.url()}\n`'```
```

That makes the command line a little bit harder to read, but the output easier to read, so... 
¯\\_(ツ)_/¯